Packs/Devo/Integrations/Devo_v2/Devo_v2.yml

```diff
@@ -115,6 +115,8 @@ script:
       defaultValue: "true"
     - name: linqLinkBase
       description: Overrides the global Devo base domain for linq linking.
+    - name: fields_to_view
+      description: A field names to include in result.
```

Comment:
**issue:** I'd like you to change the description to:
```suggestion
      description: The subset of fields (separated by a comma) that you want to display from the query result. Use this if you want to filter out unwanted columns in your result.
```
Do it for all the commands.